### PR TITLE
update CPU usage script

### PR DESCRIPTION
**Pull request by seanpray**
I've made slight changes to the CPU usage plot script with updated links from the [ci2 aws instance](https://rust-lang-ci2.s3.amazonaws.com/).

### CODE REVIEW

File: src/etc/cpu-usage-over-time-plot.sh [hunk 1]
```diff
@@ -14,13 +14,19 @@
...
+fi
+
 set -ex
 
 bucket=rust-lang-ci2
 commit=$1
 builder=$2
 
-curl -O https://$bucket.s3.amazonaws.com/rustc-builds/$commit/cpu-$builder.csv
+curl -O https://$bucket.s3.amazonaws.com/rustc-builds-alt/$commit/cpu-dist-$builder.csv
```
Comment (Mark-Simulacrum) [inline src/etc/cpu-usage-over-time-plot.sh:29]: This probably shouldn't be changing to -alt, those are less typically queried than regular builders.

File: src/etc/cpu-usage-over-time-plot.sh [hunk 2]
```diff
@@ -14,13 +14,19 @@
 #
 # Improvements to this script are greatly appreciated!
 
+if [[ $# != 2 ]]; then
+	echo "expected 2 arguments, recieved $#"
+	echo "example usage: './src/etc/cpu-usage-over-time-plot.sh 0003d1a38d97b12a389bd1e5b49f9db92094ee4a x86_64-linux-alt'"
```
Comment (Mark-Simulacrum) [inline src/etc/cpu-usage-over-time-plot.sh:19]: A non-alt builder name is probably better here.

**Comment by JohnCSimon** [comment]
Ping from triage: 
@notseanray - can you please address the comments from .Mark-Simulacrum and fix the broken build? thank you

**Comment by notseanray** [comment]
I made incorrect assumptions about cpu-$name being changed to cpu-dist-$name and -alt. I apologize as I did a poor job of checking things, sorry for any wasted time.

**Comment by Mark-Simulacrum** [comment]
r=me with commits squashed into one

**Comment by JohnCSimon** [comment]
my mistake 
@rustbot author 

@notseanray can you please squash the commits and then change to review with `@rustbot ready` ?
thank you

**Pull request merged**